### ISSUE TITLE
Motorcycle icon

### ISSUE THREAD

**Issue by mhosman** [lucide-icons/lucide#915]
## Icon Request

* Icon name: Motorcycle
* Use case: In order to complete transportation icons
* _Screenshots_ of similar icons:  https://freesvg.org/img/motorcycleicon.png

**Comment by karsa-mistmere** [comment]
This would be pretty neat to have, but I'm afraid it's been proving to be extremely tricky; I have already given it several goes on multiple occasions spanning about a year and I have yet to have a breakthrough moment. 😅

**Comment by jguddas** [comment]
Related stuff that I'm also not happy with.
![image](https://user-images.githubusercontent.com/25524993/214593616-2c84b4a6-4c5f-4dcd-bab8-9df749b809db.png)

**Comment by mhosman** [comment]
@jguddas that looks more like a bike so yeah, maybe we need another one for motorcycle.

**Comment by xnousnow** [comment]
![motorcycle](https://user-images.githubusercontent.com/119923091/229097968-60600a4c-a606-451c-8295-1de7214b210d.svg)
Proposal: duotone motorcycle
It was hard to put motorcycle and person at the same time, That's why I made this icon in duotone.
Comments appreciated!

**Comment by karsa-mistmere** [comment]
It surely is difficult to design, but at the present moment we don't do duotone icons or fills, I'm afraid. :(

**Comment by danielbayley** [comment]
@mhosman @xnousnow @karsa-mistmere @jguddas See also `motor-racing-helmet` in https://github.com/lucide-icons/lucide/pull/1304#issuecomment-1557172910

**Comment by ericfennis** [comment]
What do you think of this?
<img width="89" alt="image" src="https://github.com/lucide-icons/lucide/assets/11825403/3d049cc4-8f48-49f3-9e22-418eec889a00">


100%:
![Frame 138](https://github.com/lucide-icons/lucide/assets/11825403/6d84c127-5e18-4459-8570-7cda7d7da798)

**Comment by xnousnow** [comment]
@ericfennis That kind of looks like cicada, but I think that's best we could. :confused:

**Comment by ericfennis** [comment]
@xnousnow Lol 😆 , always interesting to know how other people see icons.

Another variation, I think this one is a bit better:
<img width="234" alt="image" src="https://github.com/lucide-icons/lucide/assets/11825403/92476390-a062-45df-affa-4afc09d14ff6">

100%
![Frame 139](https://github.com/lucide-icons/lucide/assets/11825403/1c8f9cf7-b049-4a31-a611-44111e0e4572)

**Comment by xnousnow** [comment]
@mhosman What about that?

**Comment by danielbayley** [comment]
@mhosman @xnousnow @ericfennis @karsa-mistmere @jguddas

Surely, based on [`bike`](https://lucide.dev/icons/bike), something along these lines…? 

![preview](https://github.com/lucide-icons/lucide/assets/7797479/4bf30818-14cd-4757-bda3-8cf6c6c7fc05)

https://lucide-studio.vercel.app/?value=%3Csvg++xmlns=%22http://www.w3.org/2000/svg%22++width=%2224%22++height=%2224%22++viewBox=%220+0+24+24%22++fill=%22none%22++stroke=%22currentColor%22++stroke-width=%222%22++stroke-linecap=%22round%22++stroke-linejoin=%22round%22%3E++%3Cpath+d=%22M12+17.5V14l-3-3+4-3+2+3h2%22+/%3E++%3Cpath+d=%22M19.8+8.2A3.488+3.488+0+0+1+21+11%22+/%3E++%3Cpath+d=%22m6+11-4-1%22+/%3E++%3Ccircle+cx=%2215%22+cy=%225%22+r=%221%22+/%3E++%3Ccircle+cx=%2218.5%22+cy=%2217.5%22+r=%223.5%22+/%3E++%3Ccircle+cx=%225.5%22+cy=%2217.5%22+r=%223.5%22+/%3E%3C/svg%3E

**Comment by xnousnow** [comment]
@danielbayley That's actually genious move! We could add along @ericfennis's icon with the name `motorcycle`, `motorcycle-front` or something

**Comment by jordan808** [comment]
Ah, I found this thread right before I was about to create a pull request for this icon as part of the checklist. Sent it anyway but next time I'll check the issues to throw out some ideas first. Feel free to take a look at the PR for my take, and if it's not up to par, iterate or close it out 😅

**Comment by jordan808** [comment]
Will post my design here to keep the discussion in one place, as recommended by @ericfennis. I did some explorations and also found this to be challenging. I tried several different versions, but due to the 2px spacing rule and visual weight guidelines, it was difficult to add any detail. The final version I created is as here:
![motorcycle-1](https://github.com/lucide-icons/lucide/assets/44143999/5867815d-0c9f-4852-9824-0ed5ea305131)

And here are some alternatives (in increasing detail):
![motorcyle-alt1](https://github.com/lucide-icons/lucide/assets/44143999/acc12529-c606-4016-8bc4-5e7ba09c73ad)
![motorcycle-alt2](https://github.com/lucide-icons/lucide/assets/44143999/30103f76-eeba-4020-b5db-2b628ac54b02)
![motorcycle-alt3](https://github.com/lucide-icons/lucide/assets/44143999/28d139b4-5c5d-4bab-92ad-236390617dcd)
![motorcycle-alt4](https://github.com/lucide-icons/lucide/assets/44143999/c3c516fb-c532-4346-95ec-6bfb222e1958)

Next to other icons:
![motorcycle-1](https://github.com/lucide-icons/lucide/assets/44143999/5867815d-0c9f-4852-9824-0ed5ea305131)
![plane](https://github.com/lucide-icons/lucide/assets/44143999/7d184acd-f86f-42dd-913a-6ff0db615d92)
![fuel](https://github.com/lucide-icons/lucide/assets/44143999/c7d32295-e369-45c6-9997-dac3ed299e45)
![car](https://github.com/lucide-icons/lucide/assets/44143999/ac245c64-8145-4e6a-860f-e84afdbcbdff)

If we can't get a side view to work, I think could more easily complete a set of transportation icons in the meantime from the front view like @ericfennis suggested in https://github.com/lucide-icons/lucide/issues/915#issuecomment-1898259487 , which could extend train-front, tram-front, car-front, etc. (And I agree that the left one looks like a cicada, but right one is great)

**Comment by mhosman** [comment]
I think the best way to do this is just to create a motorcycle helmet:

![ezgif-1-02fbbaf845](https://github.com/lucide-icons/lucide/assets/5334175/03f6bbe8-2132-4a42-b0c1-cdd815eb93a0)

**Comment by danielbayley** [comment]
> I think the best way to do this is just to create a motorcycle helmet:

@mhosman Already done… See `motor-racing-helmet` in https://github.com/lucide-icons/lucide/pull/1304#issuecomment-1557172910.

**Comment by mhosman** [comment]
@danielbayley why this is still not available in the latest release?

**Comment by jguddas** [comment]
> @danielbayley why this is still not available in the latest release?

Missing use case, not following the PR template.

**Comment by danielbayley** [comment]
> > @danielbayley why this is still not available in the latest release?
> 
> Missing use case, not following the PR template.

@mhosman @jguddas Yeah, the newer PR template was added after almost all of my prior PRs were opened, then applied retroactively… I didn’t have the bandwidth to go over them all again to fill all of that in… So I think there are still a lot of decent icons in limbo, until maybe eventually I can circle back around to them…

**Comment by karsa-mistmere** [comment]
@danielbayley: just to clarify, providing use cases for icon requests had always been a requirement. This requirement wasn't "applied retroactively", we just weren't enforcing it as strictly as we are now.

**Comment by ericfennis** [comment]
Love these one, well done @jamiemlaw !
My question would which icon represents the best for "motorcycle".
For example for the usecase: Navigation app and select the motorcycle for the way of transportation.

**Comment by jamiemlaw** [comment]
I would say the third one, `motorbike`. That's the most stereotypical.

But I wouldn't be surprised if `e-bike` also became its own icon, for use on delivery apps, to show how far away a driver is.

**Comment by TheMikeyRoss** [comment]
Yeah where I live (Bahrain) delivery motorcycles look like this

![Image](https://github.com/user-attachments/assets/f360c381-0213-46a2-b493-a13c7e49bda9)

But I've seen online that other countries (mostly East Asia I think) have these kind of delivery motorcycles:

![Image](https://github.com/user-attachments/assets/bbe32ddf-401d-4261-96fe-6a29eff67049)

**Comment by ericfennis** [comment]
@jamiemlaw Can you make a PR for `motorbike` and `dirtbike`? These ones seem to fit well with the use cases.
